src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -492,7 +492,7 @@ public static String injectContextDependentValuesForTest(String content) {
                                StringHelper.truncateLongId(TestProperties.TEST_ADMIN_ACCOUNT))
                       .replace("<!-- nexthour.date -->", TimeHelper.formatDate(TimeHelper.getNextHour()))
                       .replace("<!-- now.datetime -->", TimeHelper.formatTime12H(now))
-                      .replace("<!-- now.datetime.comments -->", TimeHelper.formatDateTimeForComments(now));
+                      .replace("<!-- now.datetime.comments -->", TimeHelper.formatDateTimeForComments(now, 8));
```

Comment:
This line is added to change `[${datetime.now}]` to `[${datetime.now} +0800]` in test outputs.